### ISSUE TITLE
Specified SSH key is not eligible for claiming. Only RSA and Ed25519 keys are supported for proof generation.

### ISSUE THREAD

**Issue by GalaxySciTech** [fluencelabs/dev-rewards#7]
Specified SSH key is not eligible for claiming. Only RSA and Ed25519 keys are supported for proof generation.

**Comment by GalaxySciTech** [comment]
i have available claimable , but when i generate my ssh key to claim , i can not get the coin

**Comment by nishuzumi** [comment]
Only old key have eligible. Create new key to claim is not working

**Comment by GalaxySciTech** [comment]
> Only old key have eligible. Create new key to claim is not working

so weird rule, i can not find the old key , stupid airdrop

**Comment by CodingJzy** [comment]
别搞了，领取之后还需要缴纳手续费，以太坊gas那么贵，刚刚试了一下，25u。5000代币值不值这个手续费还是个未知数

**Comment by densmirnov** [comment]
Got exaclty the same error on mac (either via local or python installation). The keys installed are RSA, used in Github

**Comment by elshan-eth** [comment]
please, find yourself in metadata.json and check that you are using the correct key

**Comment by GalaxySciTech** [comment]
> metadata.json

username is true, why you close my ticket?

**Comment by GalaxySciTech** [comment]
this case still not solve , but dev closed this ticket , fuck the project

**Comment by leo-web3** [comment]
Yes, there is obviously a problem with the verification mechanism, and developers are closing issues randomly.

**Comment by 106umao** [comment]
I also encountered this problem. I tried changing various postures many times but nothing worked. I opened the issue list of this warehouse and found no one discussing it. After searching through the browser, I found out that it was closed. What does the maintainer of this project mean?

**Comment by ExtremelySunnyYK** [comment]
facing the same issue here! just created my ssh keys

**Comment by alirezanet** [comment]
really!? last month I removed all of my old SSH keys since I didn't know which ones were still active :) now I can't verify my account 😒
